README.md

```diff
@@ -288,6 +288,7 @@ On Linux find executable file `./darknet` in the root directory, while on Window
 * To calculate anchors: `darknet.exe detector calc_anchors data/obj.data -num_of_clusters 9 -width 416 -height 416`
 * To check accuracy mAP@IoU=50: `darknet.exe detector map data/obj.data yolo-obj.cfg backup\yolo-obj_7000.weights`
 * To check accuracy mAP@IoU=75: `darknet.exe detector map data/obj.data yolo-obj.cfg backup\yolo-obj_7000.weights -iou_thresh 0.75`
+* To save the cropped image of the object detected: [`uncomment the following lines of code`](https://github.com/shubham-shahh/darknet/blob/95ec9bd9249a7c79bf2fea06dee964365f2fb61b/src/image_opencv.cpp#L980-L996)
```

Comment:
this line in readme is pointing to your repo code. This has to point to alexeyab repo, even if the link is not going to work until merged with master

Reply:
I'll update that right away. thanks

Reply:
> this line in readme is pointing to your repo code. This has to point to alexeyab repo, even if the link is not going to work until merged with master

Updated the link as mentioned.

Reply:
yes but pointing to a specific commit sha might not be the best option. Better pointing to the master trunk.

Reply:
Yes, makes sense. I committed the changes you mentioned. thanks once again